DEVELOPMENT.md

```diff
@@ -0,0 +1,19 @@
+# Development for Armadietto
+
+## Setup
+
+1. [Generate an SSH key pair](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) (if you don't already have one). It must be RSA or, preferably, ed25519.
+2. Add the *public* key [to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account).
```

Comment:
I don't think GitHub account setup is something to document specifically for Armadietto.

Reply:
Without those steps, both `yarn install` and `npm ci` fail because `git ls-remote --tags --heads ssh://git@github.com/standard/eslint-config-standard.git` fails with errors:
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

If there's a way to retrieve `eslint-config-standard.git` without using SSH, we should switch to that.  That would eliminate the need for steps 1 and 2.

Reply:
Already opened a PR for that just now: #71